go.mod

```diff
@@ -33,3 +33,9 @@ require (
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	gopkg.in/yaml.v2 v2.3.0
 )
+
+replace github.com/anchore/stereoscope => ../stereoscope
+
+replace github.com/anchore/syft => ../syft
+
+replace github.com/anchore/go-testutils => ../go-testutils
```

Comment:
ditto replace comment in other PR

Reply:
[ditto](https://github.com/anchore/syft/pull/178#discussion_r494362831)